### PR TITLE
Conveyancer confirms identity checks complete us109

### PR DESCRIPTION
**Pull request by endamccormack**

### CODE REVIEW

**Comment by endamccormack** [comment]
Also added proper logging and made the schema tests more clear

screen shot 2016-01-18 at 11 06 32

File: integration_tests/deed/test_deed.py [hunk 1]
```diff
@@ -16,30 +16,33 @@ def tearDown(self):
...
-                    "middle_name": "ann",
-                    "surname": "bloggette",
-                    "gender": "Female",
-                    "address": "Test Address With Postcode, PL14 3JR",
-                    "dob": "23/01/1986",
-                    "phone_number": "07502154061"
-                }
-            ]
-        }
+        valid_deed = json.loads(
```
Comment (Phemus) [inline integration_tests/deed/test_deed.py]: This is the old way of declaring JSON objects need to get rid of those quotes

File: application/deed/model.py [hunk 2]
```diff
@@ -10,6 +10,8 @@ class Deed(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     token = db.Column(db.String, nullable=False)
     deed = db.Column(JSON)
+    # TODO no wifi, check how to limit characters
```
Comment (danielmarker) [inline application/deed/model.py]: Found a todo here.

**Comment by endamccormack** [comment]
Good point @Kirosoft

File: application/deed/deed-api.json [hunk 3]
```diff
@@ -184,6 +185,10 @@
                 "md_ref": {
                     "pattern": "^e-MD([1-9]{5}|([1-9]{3,4}[A-Z]{1}))$",
                     "type": "string"
+                },
+                "identity_checked": {
+                  "pattern": "^[Y]$",
```
Comment (tsangsimon) [inline application/deed/deed-api.json:189]: This Regular expression only allows for Y. So are we saying that if they don't put a Y on creation on their deed application we'll reject it straight away? That seems incorrect as that check needs to only be done when either signing or making effective (need to check with BAs here)

**Comment by tsangsimon** [comment]
Logging also seems to be at info for exceptions. should be Error level for those in my opinion

**Comment by Kirosoft** [comment]
Like the addition of colour to the log messages :-)

**Comment by tsangsimon** [comment]
happy to thumbs this up after discussions and changes. 
:+1:

**Pull request merged**